frontend/templates/project/project_index.html

```diff
@@ -16,28 +16,82 @@ <h1 class="title">Projects</h1>
 		</div>
 	</section>
 
-	<section class="" id="related-projects">
+    <noscript>
+		<div id="noscript_projects" class="row">
+		  {%- for title in (titles  | sort(attribute="title")) -%}
+			<li class="col-1-4">
+			  <a href="/projects/{{ title.id }}">{{ title.title }}</a>
+			</li>
+		  {%- endfor -%}
+		</div>
+	  </noscript>
+
+	<section class="" id="related-projects" v-cloak>
```

Comment:
the `id` name is a bit misleading, `projects-overview` or just `projects` or `cards` may be better. I'm guessing this is reusing the style as defined for a single project's related projects? We should probably give it its own SCSS file.

Reply:
I would go for ``project_index_page`` to be consistent.

Reply:
went for project_index_page